src/screens/Ticketing/Purchase/Travellers/index.tsx

```diff
@@ -77,116 +92,114 @@ const Travellers: React.FC<Props> = ({navigation, route: {params}}) => {
+              />
+            ))}
+          </Sections.Section>
         )}
-      </View>
+      </ScrollView>
 
-      <View style={styles.buttons}>
-        <ThemeText type="body__link" style={styles.informationLink}>
-          Informasjon og vilkår
-        </ThemeText>
-        <Button
-          mode="primary"
-          text="Betal med Vipps"
-          disabled={isSearchingOffer}
-          accessibilityLabel="Trykk for å betale billett med Vipps"
-          icon={Vipps}
-          iconPosition="right"
-          onPress={payWithVipps}
-          style={styles.button}
-        />
-        <Button
-          mode="primary"
-          text="Betal med bankkort"
-          disabled={isSearchingOffer}
-          accessibilityLabel="Trykk for å betale billett med bankkort"
-          icon={CreditCard}
-          iconPosition="right"
-          onPress={payWithCard}
-          style={styles.button}
-        />
-        <Button
-          mode="secondary"
-          text="Avbryt"
-          accessibilityLabel="Trykk for å avbryte billettkjøp"
-          icon={Close}
-          onPress={closeModal}
-          style={styles.button}
-        />
+      <View style={styles.footer}>
+        <View style={styles.totalContainer}>
+          <View style={styles.totalContainerHeadings}>
+            <ThemeText type="body">Totalt</ThemeText>
+            <ThemeText type="label" color={'faded'}>
+              Inkl. 6% mva
+            </ThemeText>
+          </View>
+
+          {!isSearchingOffer ? (
+            <ThemeText type="heroTitle">{totalPrice} kr</ThemeText>
+          ) : (
+            <ActivityIndicator
+              size={theme.spacings.medium}
+              color={theme.text.colors.primary}
+              style={{margin: 12}}
+            />
+          )}
+        </View>
+        <View style={styles.buttons}>
+          <Button
+            mode="primary2"
+            text="Betal med Vipps"
+            disabled={isSearchingOffer}
+            accessibilityLabel="Trykk for å betale billett med Vipps"
+            icon={Vipps}
+            iconPosition="left"
+            onPress={payWithVipps}
+            viewContainerStyle={[
+              styles.paymentButton,
+              styles.vippsPaymentButton,
+            ]}
+            style={{flex: 1}}
+            textContainerStyle={{marginLeft: 20}}
+          />
+          <Button
+            mode="primary2"
+            text="Betal med bankkort"
+            disabled={isSearchingOffer}
+            accessibilityLabel="Trykk for å betale billett med bankkort"
+            icon={CreditCard}
+            iconPosition="left"
+            onPress={payWithCard}
+            viewContainerStyle={[
+              styles.paymentButton,
+              styles.cardPaymentButton,
+            ]}
+            textContainerStyle={{marginLeft: 20}}
+          />
+        </View>
       </View>
+      <View style={styles.bottomColor} />
```

Comment:
This feels a little hackish, but couldn't find a better solution on the fly. 

Originally the footer was using `{ position: absolute; bottom: 0 }` making it stick to the bottom, but this made the `<ScrollView>` for the travellers not work optimally as it did not know how much space it had available. Then I changed the footer to not be using absolute positioning, and instead use `justifyContent: 'space-between'` on the parent. But this made a gap under the footer with a different color because of the `<SafeAreaView>`, and this `<View style={styles.bottomColor} />` with absolute positioning was added just for filling the color at the bottom of the screen.

I would love to learn a better solution, if anyone has.

Reply:
We can look at this together if you want